src/auth/middleware/basic.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const base64 = require('base-64');
+const User = require('../models/users.js');
+
+module.exports = async (req, res, next) => {
+
+    if (!req.headers.authorization) { return _authError(); }
```

Comment:
if you want you can replace `return _authError();` with `res.status(403).send('Invalid Login')`